src/webgpu/shader/execution/expression/call/builtin/atanh.spec.ts

```diff
@@ -12,46 +12,39 @@ Note: The result is not mathematically meaningful when abs(e) >= 1.
 
 import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../../../gpu_test.js';
-import { kValue } from '../../../../../util/constants.js';
 import { TypeF32, TypeF16 } from '../../../../../util/conversion.js';
 import { FP } from '../../../../../util/floating_point.js';
-import { biasedRange, fullF32Range, fullF16Range } from '../../../../../util/math.js';
+import { biasedRange } from '../../../../../util/math.js';
 import { makeCaseCache } from '../../case_cache.js';
 import { allInputSources, run } from '../../expression.js';
 
 import { builtin } from './builtin.js';
 
 export const g = makeTestGroup(GPUTest);
 
-const f32_inputs = [
-  ...biasedRange(kValue.f32.negative.less_than_one, -0.9, 20), // discontinuity at x = -1
-  -1,
-  ...biasedRange(kValue.f32.positive.less_than_one, 0.9, 20), // discontinuity at x = 1
-  1,
-  ...fullF32Range(),
-];
-const f16_inputs = [
-  ...biasedRange(kValue.f16.negative.less_than_one, -0.9, 20), // discontinuity at x = -1
-  -1,
-  ...biasedRange(kValue.f16.positive.less_than_one, 0.9, 20), // discontinuity at x = 1
-  1,
-  ...fullF16Range(),
-];
+const cases = (['f32', 'f16'] as const)
```

Comment:
For this and the other changes, a comment describing what `cases` holds (non-exhaustive fields) would be good as this made me go cross-eyed trying to figure out what it was actually doing

Reply:
Do you think something like
```
// Cases: [f32|f16]_[non_]const
```

is sufficient?

This is what jzm has included for the usage of this pattern that they wrote, but I didn't replicate consistantly.